dnscrypt-proxy/example-dnscrypt-proxy.toml

```diff
@@ -435,21 +427,20 @@ cache_neg_max_ttl = 600
 
 [query_log]
 
-  ## Path to the query log file (absolute, or relative to the same directory as the config file)
-  ## Can be set to /dev/stdout in order to log to the standard output.
-
-  # file = 'query.log'
+## Path to the query log file (absolute, or relative to the same directory as the config file)
+## Can be set to /dev/stdout in order to log to the standard output.
 
+# file = 'query.log'
```

Comment:
I'm not sure reformatting these into your flavor is a good idea, and others like this ...

Reply:
Ah I see, ok thanks. will resolve it.